api/api.proto

```diff
@@ -849,17 +864,18 @@ service OpenStorageVolume {
   rpc Enumerate(SdkVolumeEnumerateRequest)
     returns (SdkVolumeEnumerateResponse) {
       option(google.api.http) = {
-        post: "/v1/volume/enumerate"
+        post: "/v1/volumes/enumerate"
```

Comment:
should be GET call

Reply:
I tried to make this one a get, but it sends data to the server (the labels in the locator), so it will not allow a GET

Reply:
ahh, I know this issue